etc/init.d/zfs.in

```diff
@@ -0,0 +1,74 @@
+# ZoL userland configuration.
+
+# Run `zfs mount -a` during system start?
+ZFS_MOUNT='yes'
+
+# Run `zfs unmount -a` during system stop?
+ZFS_UNMOUNT='yes'
+
+# Run `zfs share -a` during system start?
+# nb: The shareiscsi, sharenfs, and sharesmb dataset properties.
+ZFS_SHARE='yes'
```

Comment:
>
> @Alexqw <https://github.com/Alexqw> From what I read about that last
> comment, everything and everyone did exactly as intended.
>
@FranzUrbo I agree, with one exception: the package maintainer did not call
special attention to a significant change in behavior (and sharing data on
the network is significant in my book).

I humbly request that in the future, if such a change is made, that it be
included in the NEWS.Debian file, so that way the sysadmin has two places
to be notified about such a change.

---Alex